objectRecognitionSource/build.gradle

```diff
@@ -304,4 +304,4 @@ task copyTestResources(type: Copy, dependsOn: [configurations.testRuntime, confi
     into "$buildDir/testResources/"
 }
 
-test.dependsOn copyModels, copyTestResources
+test.dependsOn copyModels, copyTestResources
```

Comment:
do we need a `buildConnectorImage.dependsOn copyModels`?

Reply:
I removed the code that copies the testing models into the OR connector by default, so we shouldn't need it. If we want to add that feature back then we'd need it.

Reply:
OK -- missed that.